__tests__/app.test.js

```diff
@@ -15,4 +15,12 @@ describe('GET /api/categories', () => {
             expect(Array.isArray(res.body)).toBe(true);
         });
     });
+    it('returns an array of categories with slug and description properties', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
```

Comment:
I would also check the length of the response to make sure you've got all the categories